ClosedXML/Excel/XLWorkbook.cs

```diff
@@ -498,7 +498,10 @@ public void SaveAs(String file, SaveOptions options)
             else if (_loadSource == XLLoadSource.File)
             {
                 if (String.Compare(_originalFile.Trim(), file.Trim(), true) != 0)
+                {
                     File.Copy(_originalFile, file, true);
+                    File.SetAttributes(file, FileAttributes.Normal);
```

Comment:
This shouldn't be here. On which branch did you base your PR?

Reply:
You are right, I made a branch incorrectly. Only commit d5dde24 should be included in PR. I will fix it.